OpenTabletDriver.Configurations/Configurations/Huion/Kamvas Pro 19 (4k).json

```
@@ -0,0 +1,42 @@
+{
+    "Name": "Huion Tablet_GT1902",
+    "Specifications": {
+        "Digitizer": {
+            "Width": 409,
+            "Height": 230,
+            "MaxX": 81792,
+            "MaxY": 46006
+        },
+        "Pen": {
+            "MaxPressure": 16384,
+            "ButtonCount": 2
+        },
+        "AuxiliaryButtons": null,
+        "MouseButtons": null,
+        "Touch": {
+            "Width": 409,
+            "Height": 230,
+            "MaxX": 163843,
+            "MaxY": 9600
+        }
+    },
+    "DigitizerIdentifiers": [
+        {
+            "VendorID": 9580,
+            "ProductID": 107,
+            "InputReportLength": null,
+            "OutputReportLength": null,
+            "ReportParser": "OpenTabletDriver.Configurations.Parsers.Huion.GianoReportParser",
+            "FeatureInitReport": null,
+            "OutputInitReport": null,
+            "DeviceStrings": {
+                "201": "HUION_M220_240115"
```

You should not be blocking this tablet from working on minor firmware updates.
```suggestion
                "201": "HUION_M220_\\d{6}$"
```